JMitnik/haas
# Fix/broken single social media links

This PR fixes an issue where it would show a broken image url whenever one of the social media links was used as a 'single' link.

Test it out:
- Create a link CTA with a one link and select one of the social media options without uploading an image
=> Without uploading should show default picture
- Select a social media link but override the image by uploading a new one
=> New image should be displayed in client overriding the default social media image

Edit: Lately it prisma is asking me to change the foreign keys. Have you encountered that as well? Doesn't seem to harm anything so might just be them doing things a little different 😓
